Makefile

```diff
@@ -798,7 +798,11 @@ deploy/service: SSO_PROVIDER_TYPE ?= "mas_sso"
 deploy/service: REGISTERED_USERS_PER_ORGANISATION ?= "[{id: 13640203, max_allowed_instances: 5, any_user: true, registered_users: []}, {id: 12147054, max_allowed_instances: 1, any_user: true, registered_users: []}, {id: 13639843, max_allowed_instances: 1, any_user: true, registered_users: []}]"
 deploy/service: DYNAMIC_SCALING_CONFIG ?= "{developer: {compute_nodes_config: {max_compute_nodes: 3}}, standard: {compute_nodes_config: {max_compute_nodes: 9}}}"
 deploy/service: NODE_PREWARMING_CONFIG ?= "{}"
+deploy/service: ADMIN_AUTHZ_CONFIG ?= "{}"
 deploy/service: MAX_ALLOWED_DEVELOPER_INSTANCES ?= "1"
+deploy/service: ADMIN_API_SSO_BASE_URL ?= "https://auth.redhat.com"
+deploy/service: ADMIN_API_SSO_ENDPOINT_URI ?= "/auth/realms/EmployeeIDP"
+deploy/service: ADMIN_API_SSO_REALM ?= "EmployeeIDP"
```

Comment:
Thanks for updating the make file.
Can we also update the parameter section under this documentatio https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/fd1700778497d649ae4943d8d732c664f6361e28/docs/deploying-kas-fleet-manager-to-openshift.md#deploy-kas-fleet-manager